### PR TITLE
feat: make gRPC Current metric per-hostname, make sure it is decremented

### PR DESCRIPTION
**Pull request by Tasssadar**

### CODE REVIEW

File: grpc/src/main/java/com/avast/metrics/grpc/GrpcServerMonitoringInterceptor.java [hunk 1]
```diff
@@ -27,15 +28,17 @@ public <ReqT, RespT> ServerCall.Listener<ReqT> interceptCall(final ServerCall<Re
 
         cache.getMeter(method, "Calls", "count").mark();
 
-        final AtomicInteger currentCalls = cache.getGaugedValue(method, "Current");
```
Comment (Tasssadar) [inline grpc/src/main/java/com/avast/metrics/grpc/GrpcServerMonitoringInterceptor.java:30]: Ta stavajici `Current` zmizi, vadi nam to?

Comment (augi) [reply grpc/src/main/java/com/avast/metrics/grpc/GrpcServerMonitoringInterceptor.java:30]: I´m OK with this change.

**Review by augi — APPROVED** [review]
Just be aware that this change could create a lot of unique metrics in Graphite storage.

**Comment by Tasssadar** [comment]
The graphite has config that limits the retention of stats that contain the pod name, so it should be fine (plus we already have a bunch of other metrics per-pod.

**Pull request merged**